los_keeper/src/wrapper/wrapper.cc

```diff
@@ -1,26 +1,34 @@
 #include "los_keeper/wrapper/wrapper.h"
 using namespace los_keeper;
 
+Wrapper::Wrapper() {
+  obstacle_manager_ = std::make_shared<ObstacleManager>();
+  target_manager_ = std::make_shared<TargetManager3D>();
+  trajectory_planner_ = std::make_shared<TrajectoryPlanner>();
+}
+
 std::optional<Point> PlanningResult::GetPointAtTime(double t) const {
   if (!chasing_trajectory)
     return std::nullopt;
   return Point();
 }
 
-bool Wrapper::UpdateState(store::State &state) {
-  bool is_changed = false;
-  // state.is_initialized =
-  //     obstacle_manager_->IsInitialized() && target_manager_->IsInitialized();
+void Wrapper::UpdateState(store::State &state) {
+  // TODO(@): add object state, etc,.
+  state.is_data_received =
+      !obstacle_manager_->GetPointCloud().empty() && drone_state_.t_sec > 0.0;
+
```

Comment:
The ^essential^ information needed for planning is the "DroneState" and the "TargetStates."
In a scenario where there are no obstacles, the information regarding "pcl" or "structured_obstacle" does not exist.

Reply:
Okay, I will remove pcl condition